charts/rasa-x/templates/rasa-secret.yaml

```diff
@@ -11,4 +11,6 @@ data:
   rasaXToken: {{ .Values.rasax.token | b64enc | quote }}
   passwordSalt: {{ .Values.rasax.passwordSalt | b64enc | quote }}
   jwtSecret: {{ .Values.rasax.jwtSecret | b64enc | quote }}
+  initialUserName: {{ .Values.rasax.initialUser.username }}
+  initialPassword: {{ .Values.rasax.initialUser.password }}
```

Comment:
Maybe the naming is not ideal — open to suggestions :)

Reply:
I actually like it :-D